### ISSUE TITLE
make certificate compression optional

### ISSUE THREAD

**Issue by hr8** [refraction-networking/utls#160]
I dont know if its possible with a build flag or something, but I would really like to make this optional:

https://github.com/refraction-networking/utls/pull/95

I never needed this, and ever since its been added, it has bloated the package:

https://github.com/refraction-networking/utls/blob/d139a4a652b75edb89bd140bcf9e4553fc8711b1/go.mod#L6-L7

**Comment by hr8** [comment]
https://github.com/wwhtrbbtt/TrackMe/issues/11

**Comment by gaukas** [comment]
Perhaps the smarter way is to fork uTLS and take it from there. I can't think of a way to include alternative versions of go.mod`.

**Comment by hr8** [comment]
> I can't think of a way to include alternative versions of go.mod.

fair point, but couldn't you hide the feature with a build flag, like this:

https://github.com/refraction-networking/utls/blob/559ed14d97f54041561d93dff0c1e6e330929b13/fipsonly/fipsonly.go#L5

I think `go.mod` would be the same, but at least the packages would not be downloaded or built, which I believe is the case now.

**Comment by gaukas** [comment]
That's reasonable. I would welcome a PR addressing this issue. 

In fact, [ooni/utls-light](https://github.com/ooni/utls-light) might be a more suitable place to discuss the minimal uTLS.

**Comment by hr8** [comment]
thanks. I know about that option, but it wont work for me:

> focus on parrotting a single browser 

https://github.com/ooni/utls-light#utls-light

in fact, I asked @hellais last year about adding other parrots, and they are not interested to do so.
